src/TaskBoxTest/Task.js

```diff
@@ -0,0 +1,224 @@
+// Update time function
+function updateClock() {
+    const now = new Date()
+    let hours = now.getHours()
+    let minutes = now.getMinutes()
+    let seconds = now.getSeconds()
+    let meridiem = 'AM'
+
+    // convert 24 hour to 12 hour format
+    if (hours > 12) {
+        hours -= 12
+    }
+
+    // add leading 0 to hours if needed
+    if (hours < 10) {
+        hours = "0" + hours
+    }
+
+    // add leading 0 to minutes if needed
+    if (minutes < 10) {
+        minutes = "0" + minutes
+    }
+
+    // add leading 0 to seconds if needed
+    if (seconds < 10) {
+        seconds = "0" + seconds
+    }
+
+    // get AM or PM
+    if (now.getHours() >= 12) {
+        meridiem = 'PM'
+    } else {
+        meridiem = 'AM'
+    }
+
+    // format time (HH:MM:SS AM/PM)
+    const timeString = hours + ":" + minutes + ':' + seconds + ' ' + meridiem
+
+    // update html time element
+    const timeElement = document.getElementById('time')
+    timeElement.innerHTML = timeString
+}
+
+setInterval(updateClock, 1000);
+updateClock();
+
+// Update date function
+function updateDate() {
+    const now = new Date()
+
+    // formatting options
+    const options = {
+        month: 'short',
+        day: '2-digit',
+        year: 'numeric'
+    }
+
+    // convert date to format
+    const dateString = now.toLocaleDateString('en-US', options)
+
+    // update date element
+    const dateElement = document.getElementById('date')
+    dateElement.innerHTML = dateString
+}
+
+updateDate()
+setInterval(updateDate, 1000)
+
+let counter = localStorage.getItem('counter') ? parseInt(localStorage.getItem('counter')) : 0;
```

Comment:
6. localstorage could be utilized better to store tasks 
`<li>test<span>✓</span><span2>×</span2></li>` instead of this, the localstorage can only store ('list of task', 'task name') (This does **NOT** cause any bugs in the program so it's **DOESN'T** need to be urgently fixed)